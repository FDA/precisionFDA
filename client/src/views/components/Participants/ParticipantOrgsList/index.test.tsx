import React from 'react'
import { shallow } from 'enzyme'
import { QueryClient, QueryClientProvider } from 'react-query'

import { ParticipantOrgsList } from '.'


describe('ParticipantOrgsList', () => {
  it('matches snapshot', () => {
    const queryClient = new QueryClient();
    const wrapper = shallow(<QueryClientProvider client={queryClient}><ParticipantOrgsList /></QueryClientProvider>)

    expect(wrapper).toMatchSnapshot()
  })
})
