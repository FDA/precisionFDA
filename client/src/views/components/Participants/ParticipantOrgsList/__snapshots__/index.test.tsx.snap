// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ParticipantOrgsList matches snapshot 1`] = `
<ContextProvider
  value={false}
>
  <ContextProvider
    value={
      QueryClient {
        "defaultOptions": Object {},
        "mutationCache": MutationCache {
          "config": Object {},
          "listeners": Array [],
          "mutationId": 0,
          "mutations": Array [],
        },
        "mutationDefaults": Array [],
        "queryCache": QueryCache {
          "config": Object {},
          "listeners": Array [],
          "queries": Array [],
          "queriesMap": Object {},
        },
        "queryDefaults": Array [],
      }
    }
  >
    <ParticipantOrgsList />
  </ContextProvider>
</ContextProvider>
`;
